.github/workflows/python.yml

```
@@ -0,0 +1,109 @@
+name: python
+on:
+  pull_request:
+    paths:
+      - 'src/sentry/**/*.py'
+
+jobs:
+    check-missing-migration:
+      name: Check Missing Migration
+      runs-on: ubuntu-16.04
+
+      env:
+        PIP_DISABLE_PIP_VERSION_CHECK: on
+        SENTRY_LIGHT_BUILD: 1
+        SENTRY_SKIP_BACKEND_VALIDATION: 1
+        MIGRATIONS_TEST_MIGRATE: 0
+
+        # The hostname used to communicate with the PostgreSQL from sentry
+        DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
+
+        # Use this to override the django version in the requirements file.
+        DJANGO_VERSION: ">=1.11,<1.12"
+
+      services:
+        postgres:
+          image: postgres:9.6
+          env:
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+          ports:
+            # Maps tcp port 5432 on service container to the host
+            - 5432:5432
+          # needed because the postgres container does not provide a healthcheck
+          options: >-
+            --health-cmd pg_isready
+            --health-interval 10s
+            --health-timeout 5s
+            --health-retries 5
+
+      steps:
+        # Checkout codebase
+        - uses: actions/checkout@v1
+
+        # Python
+        #   Use `.python-version` to avoid duplication
+        #   XXX: can't actually read from .python-version because GitHub Actions
+        #   does not support our version (2.7.16)
+        #
+        #   XXX: Using `2.7` as GHA image only seems to keep one minor version around and will break
+        #   CI if we pin it to a specific patch version.
+        - name: Set up outputs
+          id: config
+          env:
+            MATRIX_INSTANCE: ${{ matrix.instance }}
+          run: |
+            echo "::set-output name=python-version::2.7"
+
+        # setup python
+        - name: Set up Python ${{ steps.config.outputs.python-version }}
+          uses: actions/setup-python@v1
+          with:
+            python-version: ${{ steps.config.outputs.python-version}}
+
+        # setup pip
+        - name: Install pip
+          run: |
+            pip install --no-cache-dir --upgrade "pip>=20.0.2"
```

```suggestion
            SENTRY_NO_VIRTUALENV_CREATION=1 make ensure-pinned-pip
```

Maybe not, put just putting it out there. I might want to rename `SENTRY_NO_VIRTUALENV_CREATION` before we do this, I didn't originally change it out of fear that some people might have dependend on that. Oh, or I could just add another env var `SKIP_ENSURE_VENV`, or even reuse some CI variables.

---

> or even reuse some CI variables.

@billyvg I can follow up with this sometime in the future, don't worry about it right now.